thorlcr/master/thdemonserver.cpp

```diff
@@ -180,6 +200,12 @@ class DeMonServer : public CSimpleInterface, implements IDeMonServer
     {
         lastReport = msTick();
         reportRate = globals->getPropInt("@watchdogProgressInterval", 30);
+        IPropertyTree *costs = queryCostsConfiguration();
```

Comment:
why is this a global method in jptree?
I might have expected this to be queryComponentConfiguration().queryPropTree("cost"); 
or for that to be in a helper, but surprised cost built into jptree.

Reply:
queryCostsConfiguration() was introduced several pull requests ago and has been already merged.  It is already being used in a number of different places.  I could move queryCostsConfiguration() to workunit.hpp or somewhere else?

Reply:
it's not a big deal - doesn't need to move now.
But feels like it doesn't belong in jlib (yes workunit would seem more appropriate).
@ghalliday - might want to comment.

Reply:
I suspect you're right Jake, but probably not worth moving now.